### PR TITLE
fix LocalQueueProfile_t

### PR DESCRIPTION
**Pull request by ticoann**
@ericvaandering, Eric, please review.

### CODE REVIEW

File: test/python/WMCore_t/WorkQueue_t/LocalQueueProfile_t.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from WMQuality.Emulators.DataBlockGenerator import Globals
 from WMQuality.Emulators.WMSpecGenerator.WMSpecGenerator import WMSpecGenerator
 from WMCore.WorkQueue.WorkQueue import localQueue
-from .WorkQueueTestCase import WorkQueueTestCase
+from WorkQueueTestCase import WorkQueueTestCase
```
Comment (ericvaandering) [inline test/python/WMCore_t/WorkQueue_t/LocalQueueProfile_t.py]: This is not correct python3. You need to add WMCore_t.WorkQueue_t. in front. In general, relative imports are bad, especially if we don't need them.

Comment (ericvaandering) [inline test/python/WMCore_t/WorkQueue_t/LocalQueueProfile_t.py]: Hopefully Jenkins will catch this. Let's see.

File: src/python/WMQuality/Emulators/DBSClient/DBSReader.py [hunk 2]
```diff
@@ -9,19 +9,19 @@
 
 class _MockDBSApi():
     """Mock dbs api"""
-    def __init__(self, args):
```
Comment (ericvaandering) [inline src/python/WMQuality/Emulators/DBSClient/DBSReader.py]: I don't see how any of the changes here relate to the other changes or to any issue.

**Comment by ericvaandering** [comment]
Yeah, "nose.failure.Failure:runTest was added. Status is error" is your import error.

File: src/python/WMQuality/Emulators/DBSClient/DBSReader.py (hunk 2 above)
Comment (ticoann) [inline src/python/WMQuality/Emulators/DBSClient/DBSReader.py]: args are dict type below line 68, but DbsApi first argument is url str. which gives the error. 
Also,  DbsApi init signature is (url="", proxy=None, key=None, cert=None, verifypeer=True, debug=0, ca_info=None, userAgent="") so I just to want to make compatible.

File: test/python/WMCore_t/WorkQueue_t/LocalQueueProfile_t.py (hunk 1 above)
Comment (ticoann) [inline test/python/WMCore_t/WorkQueue_t/LocalQueueProfile_t.py]: fixed it now.

**Comment by ticoann** [comment]
> Yeah, "nose.failure.Failure:runTest was added. Status is error" is your import error.

Power of jenkins test.:smile: Thanks

**Comment by ticoann** [comment]
test this please

**Comment by ticoann** [comment]
Hmm, the errors doesn't seems to be related to this patch.

**Comment by ticoann** [comment]
test this please

**Comment by ericvaandering** [comment]
@ticoann we have a winner.

**Comment by ticoann** [comment]
Yes, you are right, I don't know how that could affected the failing test though. Thanks, I will merge this.

**Comment by ericvaandering** [comment]
Probably just an instability we haven't noted yet.

**Pull request merged**